tests/e2e/Services/Tokens/TokensConsoleClientTest.php

```diff
@@ -94,6 +94,25 @@ public function testUpdateToken(array $data): array
         $dateValidator = new DatetimeValidator();
         $this->assertTrue($dateValidator->isValid($token['body']['expire']));
 
+        // Verify JWT contains correct expiration
+        $this->assertNotEmpty($token['body']['secret']);
+        $jwtParts = explode('.', $token['body']['secret']);
+        $this->assertCount(3, $jwtParts, 'JWT should have 3 parts');
+
+        // Decode JWT payload (using base64url decoding)
+        $payloadB64 = $jwtParts[1];
+        // Convert base64url to base64
+        $payloadB64 = str_replace(['-', '_'], ['+', '/'], $payloadB64);
+        // Add padding if needed
+        $payloadB64 .= str_repeat('=', (4 - strlen($payloadB64) % 4) % 4);
+        $payload = json_decode(base64_decode($payloadB64), true);
```

Comment:
Why not use JWT [decode](https://github.com/adhocore/php-jwt/blob/ad417603d9d45578b6af2089ad5b78f101c82367/src/JWT.php#L152)?